communication/socket/sockethandler.cpp

```diff
@@ -0,0 +1,394 @@
+#include "sockethandler.h"
+#include "communicationutils.h"
+#include <rapidjson/document.h>
+#include <stdio.h>
+#include <string.h>
+#include <sstream> 
+#include <iostream>
+#include <vector>
+#include <map>
+#include <stdlib.h>
+#include <sys/socket.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <netinet/tcp.h>
+#include <unistd.h>
+#include <fcntl.h>
+
+using namespace rapidjson;
+
+namespace RdkShell
+{
+  #define READ_BUFFER_SIZE 2096
```

Comment:
All RDKShell constants should start with `RDKSHELL_`.  Please rename to `RDKSHELL_READ_BUFFER_SIZE`.